requirements-dev.txt

```diff
@@ -1,10 +1,10 @@
 -r requirements.txt
-coverage==4.1
-flake8==2.6.0
 flake8-debugger==1.4.0
 flake8-quotes==0.3.0
-nose==1.3.7
+flake8==2.6.0
 pep8-naming==0.3.3
+coverage==4.1
+nose==1.3.7
```

Comment:
You've already added these as part of `requirements-test.txt` and `requirements-lint.txt`. To avoid duplication, it's better to remove everything from here and require those files with

```
-r requirements-lint.txt
-r requirements-test.txt
```